airbyte-integrations/connectors/source-pendo-python/source_pendo_python/streams.py

```diff
@@ -185,10 +185,36 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 class Report(PendoPythonStream):
     name = "report"
 
+    @property
+    def use_cache(self) -> bool:
+        return True
+
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
         return "report"
 
 
+class ReportResult(HttpSubStream, PendoPythonStream):
+    name = "report_result"
```

Comment:
is this the name of the stream that gets set in Airbyte?  if so, does it automatically get sufficed with the report id?  (i.e. the think this needs to be something like `name = "report_result_${reportId"`?)